docs/requirements.txt

```diff
@@ -3,6 +3,7 @@ furo
 jinja2
 matplotlib
 myst-parser
+pytest
```

Comment:
Why do we need pytest for the docs?

Reply:
You sure? We already have `pytest` as a dependency for tests. In case somebody is crazy enough to import `metatrain.utils.testing` (this is not something that you can do with CLI hypers, as opposed to `wandb`), I think `No module named pytest` is useful enough, no?

This always happens, in all packages there's always a `tests` folder that uses `pytest`, but you never think of it as something that people will import outside of tests.

Reply:
But we need also `pytest` not for the tests folder but also for parts of the core library. I see your point, but it feels a bit inconsistent to have a dangling dependency in the core part that is not listed in the pyproject and doesn't throw a reasonable error.